includes/Infrastructure/ServiceContainer/LazilyInstantiatedService.php

```diff
@@ -55,8 +55,7 @@ public function __construct( callable $instantiation ) {
 	 * @return Service Properly instantiated service.
 	 */
 	public function instantiate() {
-		$instantiation = $this->instantiation; // Because uniform variable syntax not supported in PHP 5.6.
-		$service       = $instantiation();
+		$service       = $this->instantiation();
```

Comment:
There's a lint issue here now (alignment).

Additionally, it displays `Call to an undefined method Google\Web_Stories\Infrastructure\ServiceContainer\LazilyInstantiatedService::instantiation()` as a check failure. Should something be changed in the lints, too, for this?

Reply:
Not sure I understand this comment. Can you give me some more detail please?

Reply:
Sorry for the confusion, see the lint failure: https://github.com/google/web-stories-wp/pull/7969/checks?check_run_id=2847956396#step:10:15

Reply:
Maybe this?


```suggestion
		$service = ($this->instantiation)();
```

Reply:
I have removed the change. It is clear to me this change is adding adding value to us.